src/recon.rs

```diff
@@ -4506,15 +4497,14 @@ pub(crate) unsafe fn rav1d_backup_ipred_edge<BD: BitDepth>(
     let y: *const BD::Pixel = (f.cur.data.data[0] as *const BD::Pixel)
         .offset((x_off * 4) as isize)
         .offset((((t.b.y + f.sb_step) * 4 - 1) as isize * BD::pxstride(f.cur.stride[0])) as isize);
+    let ipred_edge_off = (f.ipred_edge_off * 0) + (sby_off + x_off * 4) as usize;
+    let n = (4 * (ts.tiling.col_end - x_off)).try_into().unwrap();
```

Comment:
```suggestion
    let n = 4 * (ts.tiling.col_end - x_off) as usize;
```

I only used `.try_into().unwrap()` previously for a checked cast since we were blindly using it as a length without bounds checking.